apps/web-giddh/src/app/userDetails/components/subscriptions/subscriptions.component.html

```diff
@@ -123,7 +123,13 @@ <h2 class="section-head pb-0">{{seletedUserPlans?.planDetails?.name}}</h2>
 
                                             <tr>
                                                 <th>Subscription Price</th>
-                                                <td>{{seletedUserPlans?.planDetails?.amount | number}}</td>
+                                                <td>
+                                                    <!-- {{seletedUserPlans?.planDetails?.amount | number}} -->
+                                                    <span class="d-inline-flex">
+                                                        <amount-field [amount]="seletedUserPlans?.planDetails?.amount" [currencySymbol]="false" [currencyCode]="false">
```

Comment:
Add a new input in amount-field to apply/unapply giddhCurrency Pipe, like in this case it shouldn't be applied.

Reply:
Done